src/drawers/areaDrawer.ts

```diff
@@ -78,10 +78,13 @@ export module _Drawer {
       if (attrToProjector["fill"]) {
         this._areaSelection.attr("fill", attrToProjector["fill"]); // so colors don't animate
       }
-      step.animator.animate(this._areaSelection, attrToProjector);
+      if (attrToProjector["class"]) {
+        this._areaSelection.attr("class", attrToProjector["class"]);
+        this._areaSelection.classed(Area.AREA_CLASS, true);
+        delete attrToProjector["class"];
```

Comment:
My understanding of this (correct me if I'm wrong) is that the class attribute was only applied to the initial step, and was wiped out by the next step when an animation happens.